lib/php/lib/Transport/TBufferedTransport.php

```diff
@@ -22,6 +22,7 @@
 
 namespace Thrift\Transport;
 
+use Thrift\Exception\TTransportException;
```

Comment:
I don't see where TTransportException is used in the changes you made.
Is a "use" clause needed for the @inheritdoc directive to work?

Reply:
some function has doc comment `@throws TTransportException`, No one (and IDE) knows which TTransportException, so I think we should add use stmt.

Or, We always use full classname at doc comment, is it?

Reply:
My question was more for learning, and whether or not the "use" is needed for that to work.  I think I understand it now.